src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/AttendanceService.kt

```diff
@@ -30,16 +30,10 @@ class AttendanceService(
       val nomisCourseActivityId = mappingService.getMappings(attendanceSync.activityScheduleId).nomisCourseActivityId
         .also { telemetryMap["nomisCourseActivityId"] = it.toString() }
 
-      val nomisAttendanceStatus = if (attendanceSync.status == "LOCKED") {
-        getNomisAttendanceStatus(nomisCourseActivityId, attendanceSync)?.eventStatus
-      } else {
-        null
-      }
```

Comment:
We no longer need to check the Nomis status so can remove the call to the Nomis API.